src/index.js

```diff
@@ -122,16 +133,18 @@ module.exports = {
     'no-script-url': 'error',
     'no-self-compare': 'error',
     'no-sequences': 'error',
-    'no-shadow-restricted-names': 'error',
     'no-shadow': 'off',
-    'no-spaced-func': 'off',
+    'no-shadow-restricted-names': 'error',
+    'no-spaced-func': 'error',
     'no-sync': 'error',
-    'no-this-before-super': 'error',
     'no-throw-literal': 'error',
     'no-trailing-spaces': 'error',
     'no-undef-init': 'error',
     'no-undefined': 'off',
-    'no-underscore-dangle': 'off',
+    'no-underscore-dangle': ['error', {
+      allow: ['_'],
```

Comment:
Reading the examples of correct code for this rule in the [docs](http://eslint.org/docs/rules/no-underscore-dangle), it looks like this might not be necessary. Have you tested without it?